bindings/netstandard/ElectionGuard/ElectionGuard.Decryption/ElectionGuard.Decryption.csproj

```diff
@@ -6,138 +6,27 @@
-    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-
-    <TreatWarningsAsErrors>True</TreatWarningsAsErrors>
-    <NoWarn>1701;1702</NoWarn>
-    <WarningLevel>4</WarningLevel>
-    <WarningsNotAsErrors>SYSLIB0004;CS0618</WarningsNotAsErrors>
-
-    <NoStdLib>false</NoStdLib>
-  </PropertyGroup>
-  
-  <PropertyGroup Label="Debug|x86" Condition=" '$(Configuration)|$(Platform)' == 'Debug|x86' ">
+  <PropertyGroup Label="Debug" Condition=" '$(Configuration)' == 'Debug' ">
     <DebugType>full</DebugType>
     <DebugSymbols>true</DebugSymbols>
-    <Optimize>false</Optimize>
-    <OutputPath>bin\x86\Debug</OutputPath>
-    <DefineConstants></DefineConstants>
-    <PlatformTarget>x86</PlatformTarget>
-    <IntermediateOutputPath>obj\x86\Debug</IntermediateOutputPath>
-    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    
-    <TreatWarningsAsErrors>True</TreatWarningsAsErrors>
     <NoWarn>1701;1702</NoWarn>
     <WarningLevel>4</WarningLevel>
-    <WarningsNotAsErrors>SYSLIB0004;CS0618</WarningsNotAsErrors>
-
-    <NoStdLib>false</NoStdLib>
+    <WarningsNotAsErrors>SYSLIB0004</WarningsNotAsErrors>
   </PropertyGroup>
 
-  <PropertyGroup Label="Release|x86" Condition=" '$(Configuration)|$(Platform)' == 'Release|x86' ">
+  <PropertyGroup Label="Release" Condition=" '$(Configuration)' == 'Release' ">
     <DebugType>pdbonly</DebugType>
-    <Optimize>false</Optimize>
-    <OutputPath>bin\x86\Release</OutputPath>
-    <DefineConstants></DefineConstants>
-    <PlatformTarget>x86</PlatformTarget>
-    <IntermediateOutputPath>obj\x86\Release</IntermediateOutputPath>
-    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-
-    <TreatWarningsAsErrors>True</TreatWarningsAsErrors>
     <NoWarn>1701;1702</NoWarn>
     <WarningLevel>4</WarningLevel>
-    <WarningsNotAsErrors>SYSLIB0004;CS0618</WarningsNotAsErrors>
-
-    <NoStdLib>false</NoStdLib>
+    <WarningsNotAsErrors>SYSLIB0004</WarningsNotAsErrors>
   </PropertyGroup>
 
-  <ItemGroup Label="C++ Built Libraries">
-    <None Name="Windows (MSVC)" Visible="false" Include="$(ElectionGuardLibs)\Windows\$(PlatformTarget)\$(Configuration)\src\$(Configuration)\*.dll*">
-      <CopyToOutputDirectory Condition="'$(OS)' == 'Windows_NT'">Always</CopyToOutputDirectory>
-    </None>
-    <None Name="Windows HACL (MSVC)" Visible="false" Include="$(ElectionGuardLibs)\Windows\$(PlatformTarget)\$(Configuration)\libs\hacl\$(Configuration)\*.dll*">
-      <CopyToOutputDirectory Condition="'$(OS)' == 'Windows_NT'">Always</CopyToOutputDirectory>
-    </None>
-    <None Name="Windows Symbols (MSVC)" Visible="false" Include="$(ElectionGuardLibs)\Windows\$(PlatformTarget)\$(Configuration)\src\$(Configuration)\*.pdb*">
-      <CopyToOutputDirectory Condition="'$(OS)' == 'Windows_NT'">Always</CopyToOutputDirectory>
-    </None>
-
-    <None Name="Windows (msys2)" Visible="false" Include="$(ElectionGuardLibs)\Windows\$(PlatformTarget)\$(Configuration)\src\*.dll*" CopyToOutputDirectory="Always" />

```

Comment:
we dont need to copy the native libraries for these intermediate dll's. we only need them for projects that actuall "run"